cohort_3/curriculum/maestro-website/README.md

```diff
@@ -0,0 +1,106 @@
+# Maestro + webpage workshop
+
+## Website bootstrap
+
+- Open a cloud9 workspace and create a new folder titled `maestro-website`
+- Inside the folder create a new file called `index.html`
+- Make a basic website template with a `<head>` and `<body>`
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+
+  </head>
+  <body>
+
+</body>
+</html>
+```
+
+- Inbetween `<head>` tags, add `<script>` tags with an attribute `src` that is
+  equal to `"https://maestro.ngrok.com/static/baton.js"`
+- Open your website, right click on the page, click 'inspect element', and click
+  'console'
+  - If there are errors, change `https` => `http` in
+    the URL 
+- In the console, type in `maestro.Twilio.sendSms('415-555-5555','How is your
+  day?')`, replacing the phone number with your own phone number
+- Below the `<script>` tag add a second `<script>` tag in your `<head>` with the
+  `src="script.js"`
+- Create a new file in Cloud9 and name it `script.js`
+
+## Texting with programming for fun and profit
+
+### Input boxes
+
+- In the body, add two `<input>` tags (one for recipient phone number, one for
+  message)
+  - `<input>` tags are self-closing, so don't add a closing `</input>` to the
+    end
+- Give one input an `id` attribute equal to `"phone-number"` and the other input
+  an `id` of `"message"`
+
+### Button that does something when clicked
+
+- In the body, add a `<button>Click me</button>` tag
```

Comment:
Might it be a bit clearer to say something along the lines of "Under the two inputs we created, create another tag for `button`?"